src/PhpSpec/CodeAnalysis/AccessInspector.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace PhpSpec\CodeAnalysis;
+
+use ReflectionMethod;
+use ReflectionProperty;
+
+class AccessInspector implements AccessInspectorInterface
```

Comment:
What will make the Mage one different, maybe that'll give an idea what makes this version unique